webui/src/pages/user/user-namespace-extension-list.tsx

```diff
@@ -0,0 +1,107 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+import * as React from 'react';
+import { Theme, createStyles, WithStyles, withStyles, Typography } from '@material-ui/core';
+import { Namespace, isError, Extension, ErrorResult } from '../../extension-registry-types';
+import { UserNamespaceExtensionListItem } from './user-namespace-extension-list-item';
+import { ExtensionRegistryService } from '../../extension-registry-service';
+import { ErrorResponse } from '../../server-request';
+import { DelayedLoadIndicator } from '../../custom-mui-components/delayed-load-indicator';
+
+const extensionListStyles = (theme: Theme) => createStyles({
+    container: {
+        padding: theme.spacing(5, 0),
+    },
+    extensions: {
+        display: 'grid',
+        gridTemplateColumns: `repeat(auto-fit, minmax(200px, 1fr))`,
+        gap: `1.5rem`,
+        marginTop: '1rem',
+    }
+});
+
+
+class UserNamespaceExtensionListComponent extends React.Component<UserNamespaceExtensionListComponent.Props, UserNamespaceExtensionListComponent.State> {
+
+    constructor(props: UserNamespaceExtensionListComponent.Props) {
+        super(props);
+
+        this.state = {
+            loading: true,
+            extensions: undefined,
+        };
+    }
+
+    componentDidMount() {
+        this.initExtensions();
+    }
+
+    async initExtensions() {
+        const extensionsURLs: string[] = Object.keys(this.props.namespace.extensions).map((key: string) => this.props.namespace.extensions[key]);
```

Comment:
Use `Object.values(this.props.namespace.extensions)`

Reply:
Thanks, @spoenemann I tried this before but TypeScript gives me the following error:

```ts
Property 'values' does not exist on type 'ObjectConstructor'.ts(2339)
```

Adding `es2017` to the `lib` array in `base.tsconfig.json` does make the compiler happy but the TypeScript Language Server is still complaining. 
![image](https://user-images.githubusercontent.com/46004116/81837701-9d7e8380-955e-11ea-94c9-addcffb298e6.png)

```json
"lib": ["es6", "es2017", "dom"],
```

Reply:
Should I ignore the red squiggly lines and use `Object.values` after editing the config anyway? Thanks.

Reply:
No, let's leave it for now.